packages/@okta/vuepress-site/docs/guides/sign-users-out/sign-out-of-your-app/react/localsignout.md

```diff
@@ -0,0 +1,29 @@
+Call the `logout` method on the [Auth](https://github.com/okta/okta-oidc-js/tree/master/packages/okta-react#authlogouturi) object. The [withAuth](https://github.com/okta/okta-oidc-js/tree/master/packages/okta-react#withauth) [higher-order component](https://reactjs.org/docs/higher-order-components.html) makes this object easily available within components as `this.props.auth`.
```

Comment:
.... makes this object easily available within components such as `this.props.auth`.

Reply:
In this case `this.props.auth` isn't an example, it is literally the variable it is exposed as.

Reply:
How about "easily available within your components". (We continue to show you exactly how to use it)

Reply:
> In this case `this.props.auth` isn't an example, it is literally/exactly the variable name we expose.

Okay, so the word "as" is throwing me off....is there another way to say it?

...makes this object easily available within `this.props.auth` components???